src/main/features/dashboard/views/macro/claimStatus/offerAccepted.njk

```diff
@@ -32,25 +29,17 @@
 {% endmacro %}
 
 {% macro offerAcceptedForDefendantClaimDetails(claim) %}
-  {% if claim.settlement.isThroughAdmissions() %}
-    <h2 class="heading-small">{{ t('The claimant has accepted your repayment plan') }}</h2>
-    <p>
-      {{ t('{{ claimantName }} has accepted your plan and asked you to sign a legal agreement that formalises it. The agreement is called a \'settlement agreement\'.',
-      { claimantName: claim.claimData.claimant.name }) }}
-    </p>
-    <p>
-      {{ t('If you sign the settlement agreement, they can’t request a CCJ against you unless you break the terms.') }}
-    </p>
-  {% else %}
-    <h2 class="heading-small">{{ t('Settle out of court') }}</h2>
-    <p>
-      {{ t('The claimant has accepted your offer and signed a legal agreement. You need to sign the agreement to settle out of court.') }}
-    </p>
-    <p>
-      {{ t('When you’ve both signed the agreement, the claim won’t proceed.') }}
-    </p>
-    <p>
-  {% endif %}
-  {{ signAgreementLink(claim) }}
-  </p>
+  <div class="status-block">
+    {% if claim.settlement.isThroughAdmissions() %}
+      <h2 class="heading-small">{{ t('The claimant has accepted your repayment plan') }}</h2>
+      <p>{{ t('{{ claimantName }} has accepted your plan and asked you to sign a legal agreement that formalises it. The agreement is called a \'settlement agreement\'.',
```

Comment:
Unicode 2018 and 2019 for the single quotes? `‘` `’`

Reply:
done